### ISSUE TITLE
Biome temperature issue

### ISSUE THREAD

**Issue by DamienBrittan** [BiomeBundle/BiomeBundle#175]
So i have a problem with frozen forest  . Ice is turning into water and it rains instead of snowing .

**Comment by BiomeBundle** [comment]
As mentioned on the 5 other places you posted the same thing, it's caused by Serene Seasons. There isn't much I can do about this I don't think, the change would have to be made on the OTG side and there are much bigger priorities (like updating past 1.12.2).

**Comment by DamienBrittan** [comment]
Yeah , sorry for that spam .. well , i try to fix it by myself in serene's config .

**Comment by DamienBrittan** [comment]
I asked the creator of seasons mod , he said i need to change temperature of the biome , but i have not found the config file where i can change biome i want . Could you please help ?
